ide/vsc/syntax-highligthing/README.md

```diff
@@ -0,0 +1,65 @@
+# ju README
```

Comment:
How do I install this plugin?

Reply:
It's still not on the VSC Marketplace, so you cannot install them just yet. I will ping you when I publish it.